NuGet.Config

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<packageSources>
+		<add key="NugetOfficialV3" value="https://api.nuget.org/v3/index.json" />
+		<add key="Local" value="tools/LocalNugetFeed" />
```

Comment:
Can we avoid adding this local package feed and instead publish the packages to our feed?

Reply:
sure I will remove it, can you provide me location and way to upload to our feed.

Reply:
Add feed https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json and those packages should be present on that feed now.

Reply:
@weshaggard I am getting the below error when I try to use the new feed
C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.tar
gets(114,5): error : The feed 'AzArtifactsFeed [https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.j
son]' lists package 'Microsoft.Azure.KeyVault.3.0.99-Private' but multiple attempts to download the nupkg have failed.
The feed is either invalid or required packages were removed while the current operation was in progress. Verify the pa
ckage exists on the feed and try again. [D:\myFork\newtooling\prMaster\src\dotnet\Mgmt.CI.BuildTools\CI\CI.BuildTasks\T
ests\CI.BuildTasks.Tests\Tests.CI.BuildTasks.csproj]

Reply:
That is a weird error that I've not seen before. I will attempt restore those packages locally to see if I hit the same issue.

Reply:
As we chatted to help avoid the need for these keyvault packages you can introduce an assembly resolver to your build tasks for .NET Framework.

See:
https://github.com/dotnet/arcade/blob/master/src/Common/AssemblyResolver.cs
https://github.com/dotnet/arcade/blob/master/src/Common/BuildTask.Desktop.cs